.github/release-drafter.yml

```diff
@@ -0,0 +1,10 @@
+---
+version-template: '$MAJOR.$MINOR'
```

Comment:
Wieso gibt es hier keine Patchversion? Passt das zu unseren Versionierung MAJOR.MINOR.PATCH?

Reply:
Ah ok, das wusste ich nicht. Ich glaub bei den anderen APIs gibt es keine patch version
Beispiel [hier](https://github.com/europace/kex-vorgang-export-api/releases)

Wollen wir das vielleicht noch angleichen? 🤔

Reply:
In der Readme verweisen wir explizit auf https://semver.org/ und das Format `MAJOR.MINOR.PATCH`. Der github flow sollte das auch so unterstützen.

Reply:
habe ich hier angepasst